ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/CompactorUtil.java

```diff
@@ -32,23 +38,44 @@
 import org.apache.hadoop.hive.metastore.api.Partition;
 import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
 import org.apache.hadoop.hive.metastore.api.Table;
+import org.apache.hadoop.hive.metastore.api.CompactionResponse;
+import org.apache.hadoop.hive.metastore.api.CompactionRequest;
+import org.apache.hadoop.hive.metastore.api.CompactionType;
+import org.apache.hadoop.hive.metastore.api.GetValidWriteIdsRequest;
+import org.apache.hadoop.hive.metastore.api.NoSuchTxnException;
```

Comment:
nit: Is the import order correct? I was expecting that `api.CompactionResponse` would be before `api.Partition` etc.

Reply:
changed the order of imports